### DISCUSSION TITLE
helm chart not installing on kublet 1.25.3

### DISCUSSION THREAD

**Discussion by terencekmchan** [longhorn/longhorn#4779] [General]
Will this be fixed in the future?

helm install longhorn longhorn/longhorn --namespace longhorn-system --create-namespace
Error: chart requires kubeVersion: >=1.18.0-0 <1.25.0-0 which is incompatible with Kubernetes v1.25.3

**Comment by innobead** [comment]
This will be fixed in the upcoming 1.4.0.
